python/hackathon_app/tests/integration/test_users.py

```diff
@@ -63,6 +63,40 @@ def test_create_user(users: Users):
     assert not users.is_created(new_user)
     users.create(new_user)
     assert users.is_created(new_user)
+    all_users = users.rows()
+    user = all_users[-1]
+    assert isinstance(user, User)
+    assert user.first_name == new_user.first_name
+    assert user.last_name == new_user.last_name
+    assert user.email == new_user.email
+    assert user.date_created == generate_date()
+    assert user.organization == new_user.organization
+    assert user.tshirt_size == new_user.tshirt_size
+
+
+def test_update_user_updates(users: Users):
+    """update(user) should modify existing users in the users sheet. The user's
+    email is used to uniquely identify a user and cannot be amended.
+    """
+    all_users = users.rows()
+    updated_user = all_users[0]
+    updated_user.first_name = "updated_first"
+    updated_user.last_name = "updated_last"
+    updated_user.organization = "updated_org"
+    updated_user.tshirt_size = "update_size"
+    users.update(updated_user)
+
+    user = None
+    all_users = users.rows()
+    for u in all_users:
+        if u.email == updated_user.email:
+            user = u
+            break
+    assert user is not None
+    assert user.first_name == updated_user.first_name
```

Comment:
you can use the `for ... else` syntax here instead of the "set a None variable, check that it's not None after the loop" pattern

```python
    all_users = users.rows()
    for u in all_users:
        if u.email == updated_user.email:
            user = u
            break
    else:
        pytest.fail("email not found")
    assert user.first_name == updated_user.first_name
```